### PR TITLE
Update machineaction.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated values for Type and status.

Additionally, the following fields are  documented 
lastUpdateDateTimeUtc
title
cancellationRequestor
cancellationComment
cancellationDateTimeUtc
requestSource

I do not have the exact description for the below ones. But we still need to add them, could you please help?

commands
troubleshootInfo
errorHResult
externalId

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5896

File: microsoft-365/security/defender-endpoint/machineaction.md [hunk 1]
```diff
@@ -59,15 +59,20 @@ ms.technology: mde
 |Property|Type|Description|
 |---|---|---|
 |ID|Guid|Identity of the [Machine Action](machineaction.md) entity.|
-|type|Enum|Type of the action. Possible values are: "RunAntiVirusScan", "Offboard", "CollectInvestigationPackage", "Isolate", "Unisolate", "StopAndQuarantineFile", "RestrictCodeExecution" and "UnrestrictCodeExecution"|
+|type|Enum|Type of the action. Possible values are: "RunAntiVirusScan", "Offboard", "Live Response" "CollectInvestigationPackage", "Isolate", "Unisolate", "StopAndQuarantineFile", "RestrictCodeExecution" and "UnrestrictCodeExecution"|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/machineaction.md]:
```suggestion
|type|Enum|Type of the action. Possible values are: "RunAntiVirusScan", "Offboard", "Live Response", "CollectInvestigationPackage", "Isolate", "Unisolate", "StopAndQuarantineFile", "RestrictCodeExecution", and "UnrestrictCodeExecution".|
```

File: microsoft-365/security/defender-endpoint/machineaction.md [hunk 2]
```diff
@@ -59,15 +59,20 @@ ms.technology: mde
...
 |scope|string|Scope of the action. "Full" or "Selective" for Isolation, "Quick" or "Full" for Anti-Virus scan.|
 |requestor|String|Identity of the person that executed the action.|
+|requestSource|string|The name of the user/application that submitted the action.
+|cancellationRequestor|String|Identity of the person that Cancelled the action.|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/machineaction.md]:
```suggestion
|cancellationRequestor|String|Identity of the person that canceled the action.|
```

File: microsoft-365/security/defender-endpoint/machineaction.md [hunk 3]
```diff
@@ -59,15 +59,20 @@ ms.technology: mde
...
 |requestorComment|String|Comment that was written when issuing the action.|
-|status|Enum|Current status of the command. Possible values are: "Pending", "InProgress", "Succeeded", "Failed", "TimeOut" and "Canceled".|
+|cancellationComment|String|Comment that was written when Cancelling the action.|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/machineaction.md]:
```suggestion
|cancellationComment|String|Comment that was written when canceling the action.|
```

File: microsoft-365/security/defender-endpoint/machineaction.md [hunk 4]
```diff
@@ -59,15 +59,20 @@ ms.technology: mde
...
+|status|Enum|Current status of the command. Possible values are: "Pending", "InProgress", "Succeeded", "Failed", "TimeOut" and "Cancelled".|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/machineaction.md]:
```suggestion
|status|Enum|Current status of the command. Possible values are: "Pending", "InProgress", "Succeeded", "Failed", "TimeOut", and "Cancelled".|
```

File: microsoft-365/security/defender-endpoint/machineaction.md [hunk 5]
```diff
@@ -59,15 +59,20 @@ ms.technology: mde
...
 |machineId|String|ID of the [machine](machine.md) on which the action was executed.|
 |machineId|String|Name of the [machine](machine.md) on which the action was executed.|
 |creationDateTimeUtc|DateTimeOffset|The date and time when the action was created.|
-|lastUpdateTimeUtc|DateTimeOffset|The last date and time when the action status was updated.|
+|cancellationDateTimeUtc|DateTimeOffset|The date and time when the action was Cancelled.|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/machineaction.md]:
```suggestion
|cancellationDateTimeUtc|DateTimeOffset|The date and time when the action was canceled.|
```

File: microsoft-365/security/defender-endpoint/machineaction.md [hunk 6]
```diff
@@ -59,15 +59,20 @@ ms.technology: mde
...
+|lastUpdateDateTimeUtc|DateTimeOffset|The last date and time when the action status was updated.|
+|title|String|Machine action title.|
 |relatedFileInfo|Class|Contains two Properties. string `fileIdentifier`, Enum `fileIdentifierType` with the possible values: "Sha1", "Sha256" and "Md5".|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/machineaction.md]:
```suggestion
|relatedFileInfo|Class|Contains two Properties. string `fileIdentifier`, Enum `fileIdentifierType` with the possible values: "Sha1", "Sha256", and "Md5".|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mjcaparas** [comment]
@msbemba 
for now, exclude:
troubleshootInfo
errorHResult
---------------


That leaves:


**externalID:**
type: String
Description: Is Id the customer can submit in the request for custom correlation


**commands:**
relevant only for action https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/run-live-response?view=o365-worldwide. 
Those are the commands the customer wants to run using this action. (I guess you can just point to there)

**Comment by msbemba** [comment]
Thank you @mjcaparas. I have updated the two fields.

**Pull request merged**